aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -271,6 +332,41 @@ Resources:
       ReceiveMessageWaitTimeSeconds: 20
       VisibilityTimeout: 15
       QueueName: <%="activities_dead-#{stack_name}"%>
+<% unless environment == :production -%>
+  Daemon:
```

Comment:
This `daemon` resource will provision a fully-managed CI server for the environment.
For rollout, we can use this to create new CI servers for each of the environments in our pipeline, or leave this temporarily disabled for our main production-pipeline environments, leaving this useful for creating parallel `adhoc` ci-managed infrastructures in the future.